engines/scumm/script.cpp

```diff
@@ -649,6 +635,30 @@ int ScummEngine::readVar(uint var) {
 			assertRange(0, var, 25, "local variable (reading)");
 		else
 			assertRange(0, var, 20, "local variable (reading)");
+#if defined(USE_ENET) && defined(USE_LIBCURL)
+		if (ConfMan.getBool("enable_competitive_mods")) {
+			if (_game.id == GID_BASEBALL2001 &&
+				_currentRoom == 4 && vm.slot[_currentScript].number == 2090  // The script that calculates hit power
+				&& readVar(399) == 1  // Check that we're playing online
+				&& var == 2  // Local var for batter's hiting power stat
+			) {
+				int swingType = vm.localvar[_currentScript][0];
+				int powerStat, powerStatModified;
+				switch (swingType) {
+					case 2:  // Line drive or grounder swing
```

Comment:
A tiny nitpick: `switch()` statement does not add an indentation level per our coding conventions. Please fix all three statements